vcu/Src/drive_by_wire.c

```diff
@@ -118,17 +119,17 @@ uint32_t EM_Enable(uint32_t event)
 {
     bool bpsState = checkBPSState();
     bool hvEnable = getHvEnableState();
-    float brakePressure = getBrakePressure();
+    // float brakePressure = getBrakePressure();
     uint32_t state = STATE_EM_Enable;
 
     if (!bpsState) {
         DEBUG_PRINT("Failed to em enable, bps fault\n");
         sendDTC_WARNING_EM_ENABLE_FAILED(0);
         state = STATE_EM_Disable;
-    } else if (!(brakePressure > MIN_BRAKE_PRESSURE)) {
-        DEBUG_PRINT("Failed to em enable, brake pressure low (%f)\n", brakePressure);
-        sendDTC_WARNING_EM_ENABLE_FAILED(1);
-        state = STATE_EM_Disable;
+    // } else if (!(brakePressure > MIN_BRAKE_PRESSURE)) {
```

Comment:
This is commented out because of the brake fluid issue at comp right? Did we resolve that issue?